### PR TITLE
Add a few more ??=

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | 6.2
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | -
| License       | MIT
| Doc PR        | -

### CODE REVIEW

**Review by derrabus — APPROVED** [review]

File: src/Symfony/Component/Form/Tests/Extension/Core/EventListener/MergeCollectionListenerTest.php [hunk 1]
```diff
@@ -34,7 +34,7 @@ abstract protected function getBuilder($name = 'name');
 
     protected function getForm($name = 'name', $propertyPath = null)
     {
-        $propertyPath = $propertyPath ?: $name;
+        $propertyPath ??= $name;
```
Comment (stof) [inline src/Symfony/Component/Form/Tests/Extension/Core/EventListener/MergeCollectionListenerTest.php:37]: what if the property path variable is the empty string ?

Comment (derrabus) [reply src/Symfony/Component/Form/Tests/Extension/Core/EventListener/MergeCollectionListenerTest.php:37]: This is a test. If the empty string were a problem, the CI should tell us. 🙂

Comment (nicolas-grekas) [reply src/Symfony/Component/Form/Tests/Extension/Core/EventListener/MergeCollectionListenerTest.php:37]: This is a test case so we'll figure out when this happens. Tests are green :)

Comment (stof) [reply src/Symfony/Component/Form/Tests/Extension/Core/EventListener/MergeCollectionListenerTest.php:37]: oh, I missed the fact it is a test. All good then.

**Review by stof — APPROVED** [review]

**Pull request merged**